client/src/components/Ticketing/ticketingmanager/Refund/RefundOrderTicket.tsx

```diff
@@ -27,128 +47,187 @@ const Refund = () => {
           },
         },
       );
-
       if (!response.ok) {
         throw response;
       }
-
-      setShow({
-        ...show,
-        showPopUp: true,
-        message: `Order ${orderID} successfully refunded`,
-        title: `Order Refund Successful`,
-        success: true,
-      });
+      setOrders((orders) => orders.filter((order) => order.orderid !== orderID));
+      setPopUp(
+        'Refund Successful',
+        `Order ${orderID} successful refunded`,
```

Comment:
Nitpick, but probably should be `successfully refunded`

Reply:
FIxed